content/strategy-goals/strategy/code-graph/search/index.md

```diff
@@ -12,17 +12,19 @@ Quicklinks:
 
 ## Mission
 
-The Search team's mission is to enable developers to find the code they need when they need it.
+The Search team's mission is to enable developers to find the code they need when they need it, and then act on it.
 
 In the near term, we want to make sure our customers are aware of and using our differentiated code search features. For example, Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results. We envision that Search will power [unified](../index.md#deliver-a-unified-experience) use cases of Sourcegraph, at any scale and across the entire OSS universe.
 
 ## Competitive landscape
 
-Our competitive landscape in the search space is sparse at the moment, but there are serious potential competitors we need to monitor–GitHub is the elephant in the room. It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of using Sourcegraph will be too high for users to switch, if GitHub’s search improves significantly.
+The biggest threat to our dominance as the leader in open-source, free, and commercially available code search is the advnet of GitHub Code search, currently widely available in technology preview. Last quarter we stated:
 
-There are also existing open source projects that may be “good enough” for many users. While we don’t believe these pose a serious threat to our enterprise business, they are competitors with our Cloud offering and, to the extent users prefer those experiences, we need to consider how to compete against them for attention and usage.
+> It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of using Sourcegraph will be too high for users to switch, if GitHub’s search improves significantly.
```

Comment:
Quick question here @benvenker @jyavorska , since I'm planning to add a code insights competitive section after our GA – do we want to be this explicit and blunt publicly in our competitive analysis? 

Are we worried at all that customers (or even GH folks) would see this and take actions that they wouldn't otherwise? (E.g., does this undermine our competitive messaging elsewhere if GH or prospects can just say "look, you say it yourself, you think GH could be good enough").

If we were to be less blunt, we could adopt at least a less pessimistic tone, and just note "It's possible" rather than "It's very likely"? That said, no expert and curious to hear your thoughts! (No urgency/not blocking either – feel free to close this PR and we can discuss this at some later time)

Reply:
I think we can be transparent even when it's hard but I agree that we can be more positive about how we plan to compete. I think we have real reason to be positive, and I'm also not sure switching costs is the key to the strategy. I left some similar comments above, but would agree we can work together to improve this section to be about how we are going to win even in a competitive environment (not based on delusion, but based on our real strengths).

Reply:
Idea: phrase this as "Sourcegraph has superior capabilities (<link to feature comparison>), but GitHub has a very powerful distribution channel so we need to evangelize Sourcegraph widely to make the value of great code search known."

Reply:
pushing some changes to these effects shortly